app/_includes/prereqs/products/operator.md

```diff
@@ -17,15 +17,17 @@
 {% if prereqs.v_maj == 1 %}
 
    ```bash
-   helm upgrade --install kgo kong/gateway-operator -n kong-system --create-namespace \
+   helm upgrade --install kgo kong/gateway-operator -n kong-system \
+     --create-namespace \
```

Comment:
@pmalek This prereq is used in several other KO how tos. Is this just a formatting change? I want to make sure it's not a breaking change that will break the other how tos.